INSTRUCTION.md

```diff
@@ -0,0 +1,36 @@
+# Instructions
+
+## Running the MySQL Container
+
+1. Build the MySQL Docker image:
+    ```sh
+    docker build -t mysql-local:1.0.0 -f Dockerfile_mysql .
+    ```
+
+2. Run the MySQL container with a volume attached:
+    ```sh
+    docker run -d --name mysql-container -v mysql_data:/var/lib/mysql -p 3306:3306 mysql-local:1.0.0
+    ```
+
+## Running the Django Application Container
+
+1. Update the `DATABASES` setting in `todolist/settings.py` to use the IP address of the running MySQL container.
+
+2. Build the Django application Docker image:
+    ```sh
+    docker build -t todoapp:2.0.0 .
+    ```
+
+3. Run the Django application container:
+    ```sh
+    docker run -d --name todoapp-container --link mysql-container:mysql -p 8080:8080 todoapp:2.0.0
```

Comment:
The `--link` flag is deprecated and should be replaced with Docker networks for better container communication. Consider creating a Docker network and connecting both the MySQL and Django containers to it.